src/main/java/uk/gov/ons/ctp/response/action/config/SampleSvc.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.ons.ctp.response.action.config;
+
+import lombok.Data;
+import net.sourceforge.cobertura.CoverageIgnore;
+import uk.gov.ons.ctp.common.rest.RestUtilityConfig;
+
+/** App config POJO for survey service access - host location and endpoint locations */
```

Comment:
I don't think this comment and it's also wrong

Reply:
Removed